tools/openshift-ci/Dockerfile

```diff
@@ -20,6 +20,10 @@ RUN set -x && \
     JQ_RE='^.*"browser_download_url": ?"(http[^"]*jq-linux64)".*$' && \
     curl -sSL $GITHUB_API_CURL_OPTS "$JQ_SPEC" | sed -En "s#$JQ_RE#\1#p" | xargs -d '\n' curl -sSL -o /usr/local/bin/jq && \
     chmod +x /usr/local/bin/jq && \
+    YQ_SPEC="https://api.github.com/repos/mikefarah/yq/releases/latest" && \
```

Comment:
Presently many step is using a specific version of yq version, but not the latest one, so I guess we can copy https://github.com/openshift/installer/blob/master/images/installer/Dockerfile.upi.ci#L46-L51 here to keep consistent.